WifiART/src/structs/Subelement.java

```diff
@@ -0,0 +1,13 @@
+package structs;
+
+/**
+ * Represents a LCI/LCR subelement
```

Comment:
nit: The interface can represent a random subelement from wifi packet, not just LCI/LCR, right?

Reply:
Yes. The interface abstracts the implementation of the encoding to a hex buffer, so any subelement can use it, not just LCI/LCR. I created the interface in anticipation of someone adding some of the remaining LCI/LCR subelements in the future.